pom.xml

```diff
@@ -893,6 +902,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
         <configuration>
+          <sourcepath>${project.build.directory}/delombok</sourcepath>
```

Comment:
The currennt cherck was on the sourcecode instead of the generated lombok code, right?

Reply:
Yes, though only for site generation. This config value is copied from the javadoc generation during normal build.